dpxdt/tools/local_pdiff.py

```diff
@@ -331,7 +331,7 @@ class NotReadyError(Exception):
         try:
             url = waitfor['url']
             r = requests.head(url)
```

Comment:
What about providing `allow_redirects=True` here? [See this documentation](http://docs.python-requests.org/en/latest/api/).

Otherwise, style nit: I'd put `(200, 301, 302)` in a tuple since it's immutable.